sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/MessageSerializerResult.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This class isn't used anywhere?

Reply:
I have deleted it.  I thought of using it but survived without it.  Will Make sure to remove it